doc/api/perf_hooks.md

```diff
@@ -656,3 +656,4 @@ require('some-module');
 
 [`timeOrigin`]: https://w3c.github.io/hr-time/#dom-performance-timeorigin
 [W3C Performance Timeline]: https://w3c.github.io/performance-timeline/
+[Async Hooks]: async_hooks.html
```

Comment:
Can you please swap this with the previous line to keep them sorted alphabetically?

Reply:
Absolutely. Just to make sure, you mean

case 1:
a. `timeOrigin`
b. `Async Hooks`
c. `W3C Performance Timeline`

or case 2
a. `Async Hooks`
b. `timeOrigin`
c. `W3C Performance Timeline`
?

Reply:
@manidlou, case 1. Thanks.